packages/semantic-release-config/index.js

```diff
@@ -1,43 +1,55 @@
 module.exports = {
   plugins: [
     [
-      "@semantic-release/commit-analyzer",
+      '@semantic-release/commit-analyzer',
       {
-        preset: "conventionalcommits",
+        preset: 'conventionalcommits',
         releaseRules: [
-          { type: "docs", release: "patch" },
-          { type: "refactor", release: "patch" },
-          { type: "style", release: "patch" },
-          { type: "test", release: "patch" },
-          { type: "ci", release: "patch" },
-          { type: "build", release: "patch" },
+          { type: 'docs', release: 'patch' },
+          { type: 'refactor', release: 'patch' },
+          { type: 'style', release: 'patch' },
+          { type: 'test', release: 'patch' },
```

Comment:
I think we should not release on `test`, then we can keep `test` as `hidden` in the `/release-notes-generator`. What do you think of this?

Reply:
Agree